ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh

```diff
@@ -37,6 +46,7 @@ function mirror_image(){
 
 function update_icsp(){
     local source_release_image_repo
+    set_proxy_env
```

Comment:
We need to unset the proxy when we don't need to access the cluster.

Reply:
ah, this function is the last one that needs proxy. so that's fine. please ignore ^